# set ARM64 flags the same way as in parent CMakeLists.txt
# 
--- source/dynamicHDR10/CMakeLists.txt.orig	2021-03-16 12:53:00.000000000 +0000
+++ source/dynamicHDR10/CMakeLists.txt	2022-03-26 13:06:56.201819727 +0000
@@ -43,7 +43,11 @@
         endif()
     endif()
     if(ARM AND CROSS_COMPILE_ARM)
-        set(ARM_ARGS -march=armv6 -mfloat-abi=soft -mfpu=vfp -marm -fPIC)
+        if(ARM64)
+            set(ARM_ARGS -fPIC)
+        else()
+            set(ARM_ARGS -march=armv6 -mfloat-abi=soft -mfpu=vfp -marm -fPIC)
+        endif()
     elseif(ARM)
         find_package(Neon)
         if(CPU_HAS_NEON)
